src/command_modules/azure-cli-network/azure/cli/command_modules/network/tests/test_network_commands.py

```diff
@@ -134,13 +134,11 @@ def body(self):
         rg = s.resource_group
         s.cmd('network public-ip create -g {} -n {} --dns-name {} --allocation-method static'.format(rg, s.public_ip_name, s.dns), checks=[
             JMESPathCheck('publicIp.provisioningState', 'Succeeded'),
-            JMESPathCheck('publicIp.publicIPAllocationMethod', 'Static'),
-            JMESPathCheck('publicIp.dnsSettings.domainNameLabel', s.dns)
```

Comment:
we should file a bug on this and comment out these lines for now

---
In reply to: [78789896](https://github.com/Azure/azure-cli/pull/875#discussion_r78789896) [](ancestors = 78789896)